third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -4509,6 +4706,136 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
         }
     }
 
+    /// Translates a call that might be a curry expression
+    fn translate_curry_or_call(
+        &mut self,
+        expected_type: &Type,
+        ftype: Type,
+        fun_exp_or_oper: FunExpOrOper,
+        arg_types: Vec<Type>,
+        translated_args: Vec<Exp>,
+        mask: u128,
```

Comment:
For the mask for closures, it should be only u64. More is not needed. u128 should not be put into an instruction, to large for it, but its not needed anyway.